scripts/pylib/pytest-twister-harness/src/twister_harness/twister_harness_config.py

```diff
@@ -43,7 +44,6 @@ def __post_init__(self):
         else:
             self.app_build_dir = self.build_dir
 
-
```

Comment:
Incongruent with PEP8, which requires two blank lines before and after class definitions.

Reply:
fixed